test/conformance/kuttl/validate/patterns/compare-arrays/role-pass.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pass
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - persistentvolumeclaims
```

Comment:
Add some more here. The expectation is more are ok but the ones in the policy are required.

Reply:
It will break, first thing is to compare arrays lengths.

Reply:
Hmm, maybe i'm wrong, let's check

Reply:
Ok, guess I was wrong.

Reply:
So strict equality is ok here ?

Reply:
I think so, otherwise an "include" should use the `^()` anchor notation.

Reply:
Ok, let's merge this one, i will add another kuttl test to make sure `^()` works correctly.

Reply:
But `^()` matches a single element 🤔

Reply:
Ah ok, I guess in that case the answer is to rewrite it as a deny rule using a JMESPath expression.

Reply:
Ok, do we want to merge or close this fix ?

Reply:
This fix is just an exact match, correct?

Reply:
Yes, i want to make sure this doesn't break `^()` though.

Reply:
I think it's fine to accept. The behavior is aligned with the current behavior, just allows multiple and not singular array entries.